cmd/down.go

```diff
@@ -7,16 +7,19 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var downCmd = &cobra.Command{
-	Use: "down",
-	Short: "Deletes the pods of the specified docker compose services. " +
-		"If all docker compose services would be deleted then the Kubernetes services are also deleted.",
-	Long: "destroy all pods and services",
-	Run:  downCommand,
+func setupDownCli() *cobra.Command {
```

Comment:
Rename to `newXXXCommand`, e.g. here: `newDownCommand`

Reply:
Done